packages/runtime/src/next/api-handler/index.ts

```diff
@@ -92,6 +92,18 @@ export function MakeswiftApiHandler(
   ): Promise<NextResponse<MakeswiftApiHandlerResponse> | void> {
     return match(args)
       .with(routeHandlerPattern, async args => {
+        const [req] = args
+        if (req.method === 'OPTIONS') {
+          return new NextResponse(null, {
+            status: 200,
```

Comment:
Funny enough, I questioned the lack of OPTIONS request handling here in my API handlers re-write, and ended up [adding it for both App and Pages Router](https://github.com/makeswift/makeswift/pull/1030/files#r2198518869), but it didn't occur to me that this might be the root cause when we were looking into this yesterday :(